libs/utils/src/http/endpoint.rs

```diff
@@ -63,8 +63,37 @@ async fn prometheus_metrics_handler(_req: Request<Body>) -> Result<Response<Body
     Ok(response)
 }
 
+pub fn add_request_id_middleware<B: hyper::body::HttpBody + Send + Sync + 'static>(
+) -> Middleware<B, ApiError> {
+    Middleware::pre(move |mut req| async move {
+        let x_request_id = HeaderName::from_static("x-request-id");
```

Comment:
Is it the same as in console?

Reply:
yes : https://github.com/neondatabase/cloud/blob/189ed5819bf6c9c35038a3462c7604585b349e28/internal/resttooling/requestid/requestid.go#L16